packages/react-native/ReactCommon/react/renderer/components/view/primitives.h

```diff
@@ -85,6 +85,31 @@ enum class BorderCurve : uint8_t { Circular, Continuous };
 
 enum class BorderStyle : uint8_t { Solid, Dotted, Dashed };
 
+// [macOS [visionOS]
```

Comment:
Slight nit: Should this be `[visionOS` instead of `[visionOS]`?

(Not blocking, by the way. This can easily be fixed in a later PR.)

Reply:
Yeaaah this double tag gets weird. I'll take a pass through them all before merging